bootfiles/bootinst.sh

```diff
@@ -40,5 +40,9 @@ if [ "$DEV" != "$PART" ]; then
    ) | fdisk $DEV >/dev/null 2>&1
 fi
 
+# UEFI boot loader
```

Comment:
Test result in https://github.com/Tomas-M/linux-live/pull/84#issuecomment-537543083:
> Files should be copied to the root directory of the usb drive. Instead they will be copied to `linux/boot/EFI/Boot`.

Proposed solution:
```
# UEFI boot loader
mkdir -p "$BOOT/../../EFI"
mv "EFI/Boot" "$BOOT/../../EFI/"
```

Reply:
Implemented this, thanks for testing!